README.md

```diff
@@ -8,15 +8,112 @@
+
+The `ballerinax/jira` package offers APIs to interact with the [Jira REST API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro), providing access to various endpoints for managing issues, filters, projects, and other resources.
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Jira connector, you must have a Jira Cloud account with API access. If you do not have one, you can sign up for a Jira account on [Atlassian’s website](https://developer.atlassian.com/cloud/jira/platform/).
+
+### Step 1: Create an API Token
+1. Login to [Atlassian Account](https://developer.atlassian.com/cloud/jira/platform/)
+
+2. Click on the profile icon to get into developer console
+
+ <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/atlassian-developer-console#1.png alt="Atlassian developer console" style="width: 70%;">
+
+3. In the new window, click on the profile again and go to 'Manage Account'
+
+4. Click and go to security tab and scroll down to the bottom to generate API token by clicking 'Create and Manage API token'
+
+ <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/create-API-Token#2.png alt="Create API Token" style="width: 70%;">
+
+5. Give a name to the API token and copy and save it in a secret place
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/generate-API-Token#3.png alt="Generate API Token" style="width: 70%;">
+
+### Step 2: Integrate JIRA for your account
+1. Click on the grid shaped icon in the right corner (Before Atlassian Account logo). Click on explore products
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/try-jira-cloud#5.png alt="Try jira" style="width: 70%;">
+
+2. Click on the try cloud button under Jira
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/integrate-Jira#4.png alt="Integrate Jira" style="width: 70%;">
+
+3.Click try jira for free
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/get-Jira-Free#6.png alt="Get jira for free" style="width: 70%;">
+
+4.Enter your email that you used previously to log into atlassian account
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/email-for-jira#7.png alt="Enter email for jira account" style="width: 70%;">
+
+5.Make sure to copy the Domain URL provided for you (Your site url)
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-jira/main/docs/setup/resources/domain-URL-Jira#8.png alt="Create API Token" style="width: 70%;">
```

Comment:
Image names are different in the resource folder. Please check all the images.

Reply:
will do